.github/workflows/test.yml

```diff
@@ -19,6 +19,8 @@ jobs:
       - run: yarn lint
       - run: yarn audit --groups dependencies
       - run: yarn test:unit --coverage
+      - run: yarn test:e2e
+      - run: yarn test:all
```

Comment:
We don't need to call these commands here as part of CI because this section of the workflow is ONLY responsible for calling the unit tests. Look elsewhere in the file to see where the e2e tests are configured. No changes are needed in this file.
```suggestion
```

Reply:
Deleted these lines of code.